src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -1329,7 +1327,7 @@ const WorkflowView = _.flow(
           key,
           accept: '.json',
           multiple: false,
-          disabled: currentSnapRedacted || !!WorkspaceUtils.editWorkspaceError(workspace) || data.length === 0,
+          disabled: currentSnapRedacted || WorkspaceUtils.canEditWorkspace(workspace)[0] ? false : true || data.length === 0,
```

Comment:
You could use `!` here instead of `?:`
```suggestion
          disabled: currentSnapRedacted || !WorkspaceUtils.canEditWorkspace(workspace)[0] || data.length === 0,
```

Reply:
https://github.com/DataBiosphere/terra-ui/pull/4268/commits/d250d450c4682c362ec74a9d9fcaf792b409f693
I thought the ternary was clearer in many cases, but since the linter removed most of those, might as well remove this one as well.